test/support/api/workspaces.js

```diff
@@ -23,6 +23,12 @@ export function getWorkspaces(data, { depth = 0 } = {}) {
   return mergeJsonApi(resource, data, { VALID: { relationships: _.keys(defaultRelationships) } });
 }
 
+const workspaces = getResource(fxTestWorkspaces, TYPE);
```

Comment:
I tried to do `const workspaces = getWorkspaces()` here but the `clinicians: getRelationship(getClinicians({}, { depth })),` was throwing an error.

Reply:
This is a dependency issue..  I don't know how to fix it because we `import` all of the files in one file..  I think we'd have to always export functions..

Reply:
So leave this how I have it for now?